Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
+
+  # lots of OS X support things were missing from this release
+  patch :DATA
```

Comment:
Yeah,  but it needs more guards before it can go upstream. As in: it needs to be versioned for OS X, so that it doesn't affect Linux and FreeBSD and others. So it is "easier" to test downstream.